### PR TITLE
Revert "Rename gray color palette name"

### PR DESCRIPTION
**Pull request by gtsiolis**
Reverts gitpod-io/gitpod#10276.

Until we update to Tailwind 3 we need to stay with this color palette as this change is changing the warm gray colors we're using to complement our branding colors and defaults to using the gray palette.

See relevant discussion (internal).

| BEFORE the color update | AFTER  the color update |
|-|-|
| light-theme-before | light-theme-after |
| dark-theme-before | dark-theme-after |

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
@gtsiolis: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by gtsiolis** [comment]
@andrew-farries could you review this since you reviewed the initial PR in https://github.com/gitpod-io/gitpod/pull/10276?

**Comment by jankeromnes** [comment]
Holding to unblock the merge queue 😇 

/hold

**Comment by jankeromnes** [comment]
Ah, sorry @gtsiolis, I believe you might have caught the `main` branch in a bad state.

Could you please do one of the following?
- 🅰️ Rebase your Pull Request (e.g. by running `git pull --rebase origin main && git push -f`)
- 🅱️ Or, do the same revert again in a new Pull Request (based on a newer, fixed `main`)

**Comment by gtsiolis** [comment]
Thanks, @jankeromnes! I noticed that builds in werft[[1](https://werft.gitpod-dev.com/)] were failing a lot today, that's why I have not rebased yet, but let me try again now.

**Comment by jankeromnes** [comment]
Worked well! 🎉

/unhold
